### PR TITLE
7840: Get catalogue order item

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#7840](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7840) 
Task: [AB#7854](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7854)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/GetOrderItemModel.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        public string EstimationPeriod { get; set; }
+
+        public string ProvisioningType { get; set; }
+
+        public string Type { get; set; }
+
+        public string CurrencyCode { get; set; }
+
+        public ItemUnitModel ItemUnitModel { get; set; }
```
Comment (CallumBradburyBJSS) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/GetOrderItemModel.cs:25]: This should be ItemUnit not ItemUnitModel, for the property name

File: tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CatalogueSolutionsControllerTests.cs [hunk 2]
```diff
@@ -543,6 +543,75 @@ public async Task UpdateOrderItemAsync_Error_ReturnsError()
...
+            result.Result.Should().BeOfType<NotFoundResult>();
+        }
+
+        [Test]
+        public async Task GetOrderItemAsync_OrderItemExists_ReturnsResult()
+        {
+            var context = CatalogueSolutionsControllerTestContext.Setup();
+
+            var serviceRecipients = new List<(string Ods, string Name)>
+
```
Comment (CallumBradburyBJSS) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CatalogueSolutionsControllerTests.cs:571]: extra line here can be removed

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/GetOrderItemModel.cs (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/GetOrderItemModel.cs:25]: 👍

File: tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CatalogueSolutionsControllerTests.cs (hunk 2 above)
Comment (RuthBerry96) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CatalogueSolutionsControllerTests.cs:571]: 👍

**Pull request merged**